pdf_build_src/process_markdowns.py

```diff
@@ -188,4 +346,7 @@ def edit_titlepage():
 
     # Step 6: remove all internal links
     remove_internal_links(duplicated_src_dir_path, 'cross')
-    remove_internal_links(duplicated_src_dir_path, 'same')
\ No newline at end of file
+    remove_internal_links(duplicated_src_dir_path, 'same')
+
+    # Step 7: correct number of dashes and fences alignment for rendering tables in PDF
+    correct_tables(duplicated_src_dir_path) 
```

Comment:
Please end all files with a blank newline. This is usually something you can set in the settings of your editor to do automatically

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file